frontends/web/src/style/fonts.css

```
@@ -1,14 +1,14 @@
 @font-face {
   font-family: 'Inter';
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)